tests/rhtap-demo/config/scenarios.go

```diff
@@ -32,7 +32,7 @@ var TestScenarios = []TestSpec{
 				AdvancedBuildSpec: &AdvancedBuildSpec{
 					TestScenario: TestScenarioSpec{
 						GitURL:      "https://github.com/redhat-appstudio/integration-examples.git",
-						GitRevision: "843f455fe87a6d7f68c238f95a8f3eb304e65ac5",
+						GitRevision: "main",
 						TestPath:    "pipelines/integration_resolver_pipeline_pass.yaml",
```

Comment:
This Spec is used by rhtap-demo test ([right here](https://github.com/redhat-appstudio/e2e-tests/blob/6f631c21e4de0cce2c2d74a3dcfec91ddaa265fc/tests/rhtap-demo/rhtap-demo.go#L352)), and I made this change because I thought it's better to use the main branch rather than a hardcoded commit. wdyt?